app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -484,111 +437,85 @@ $after-button-padding-left: govuk-spacing(4);
     box-shadow: none;
     color: govuk-colour("white");
 
+    &::after {
+      background: none;
+    }
+
     // stylelint-disable max-nesting-depth
     @media (hover: hover) and (pointer: fine) {
       &:hover {
         .gem-c-layout-super-navigation-header__navigation-top-toggle-button-inner {
-          color: govuk-colour("mid-grey");
+          color: govuk-colour("white");
```

Comment:
I think you can remove the colour style too since it's inherited from the `focus-not-focus-visible` mixin styles.

Reply:
Good spot, I've removed the colour style here